### ISSUE TITLE
Update to support 2020-10 release

### ISSUE THREAD

**Issue by andyw8** [Shopify/shopify_python_api#410]
@tanema 

With the 2020-10 release coming up soon, I expect we'll need to update https://github.com/Shopify/shopify_python_api/blob/master/shopify/api_version.py#L27 ?

(and would this also involve bumping to version 9?)

**Comment by tanema** [comment]
Yes it will need to be updated and no we should only be updating the patch version.

**Comment by mam8cc** [comment]
Is there a timeline for this client to be released?  I see that https://github.com/Shopify/shopify_python_api/pull/416 is WIP.

**Comment by andyw8** [comment]
@mam8cc It's now released: https://pypi.org/project/ShopifyAPI/8.0.4/
